ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -279,8 +279,7 @@ trait EventStorageBackend {
       endInclusive: Long,
   )(connection: Connection): Vector[EventStorageBackend.Entry[Raw.FlatEvent]]
 
-  /** Max event sequential id of observable (create, consuming and nonconsuming exercise) events. */
-  def maxEventSequentialIdOfAnObservableEvent(offset: Offset)(connection: Connection): Option[Long]
+  def maxEventSequentialIdOfAnObservableEvent(offsetInclusive: Offset)(connection: Connection): Long
```

Comment:
comment not helps much, also simplifies the return type  to move the complexity below